spaceinvaders.py

```diff
@@ -1,648 +1,687 @@
+#!/usr/bin/env python
+
 # Space Invaders
 # Created by Lee Robinson
 
-#!/usr/bin/env python
 from pygame import *
 import sys
-from random import shuffle, randrange, choice
```

Comment:
Is shuffle not being used anymore?

Reply:
It's enough 'randint'. But it was bad idea to rand all enemies in EnemiesGroup.random_bottom. One alive in column enemy has less chance to shoot. Please, look at 'Fix chance to shoot for almost dead enemy columns' (f926bf0)